common/app/conf/switches/ABTestSwitches.scala

```diff
@@ -34,4 +34,14 @@ trait ABTestSwitches {
     sellByDate = Some(LocalDate.of(2024, 9, 30)),
     exposeClientSide = true,
   )
+
+  Switch(
+    ABTests,
+    "ab-opt-out-frequency-cap",
+    "Test the Opt Out frequency capping feature",
+    owners = Seq(Owner.withEmail("commercial.dev@theguardian.com")),
+    safeState = Off,
+    sellByDate = Some(LocalDate.of(2024, 8, 30)),
```

Comment:
fyi this is a shorter expiry than the client side ab test?

https://github.com/guardian/commercial/blob/64d87b115ec766f22e777d64f1c1f609d4c55cdb/src/experiments/tests/opt-out-frequency-cap.ts#L7

Reply:
Thanks, shortened the commercial expiry